src/sagemaker/tensorflow/estimator.py

```diff
@@ -185,21 +196,54 @@ def __init__(self, training_steps=None, evaluation_steps=None, checkpoint_path=N
                     Examples:
                         123.dkr.ecr.us-west-2.amazonaws.com/my-custom-image:1.0
                         custom-image:latest.
+            script_mode (bool): If set to True will the estimator will use the Script Mode containers (default: False).
+                This will be ignored if py_version is set to 'py3'.
```

Comment:
I'm a little worried about the implicit script mode if Python 3 given the popularity of the Python 3 support for TF feature request. Maybe we should log a warning somewhere if `py_version` is set but not `script_mode`?

Reply:
Good point. I will do this in the follow up PR with the doc update